javascript-starter/index.js

```diff
@@ -0,0 +1,12 @@
+// index.js Used in development for hot reloading/nodemon
+import fs from "fs";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+fs.readdirSync(`${__dirname}/functions`)
+  .filter((file) => file.slice(-3) === ".js")
+  .forEach((file) => {
+    import(`./functions/${file}`);
+  });
```

Comment:
@davemooreuws while we're at it we should modify these to read from nitric.yaml.